python/cudf/cudf/tests/test_csv.py

```diff
@@ -1361,12 +1343,16 @@ def test_csv_writer_datetime_data(tmpdir):
 @pytest.mark.parametrize(
     "columns",
     [
-        ["Integer", "Date", "Float", "Integer2", "Category"],
-        ["Category", "Date", "Float"],
+        # Category is not yet supported from libcudf
+        # ["Integer", "Date", "Float", "Integer2", "Category"],
```

Comment:
Just as a note, we my want to simply mark these as skipped as opposed to commenting them. That will give us an indication that there is something here we may want to address later. Here's [an example]( https://github.com/dask/distributed/blob/e7ba316d496a23d073188466571ce1580336aba4/distributed/comm/tests/test_ucx.py#L177-L182 ) of how we might do this.

Reply:
To be clear, we never actually properly supported these. They would be hashed by the reader / writer which is incorrect behavior that we don't want to continue having.